x-pack/metricbeat/docs/modules/foo.asciidoc

```diff
@@ -0,0 +1,39 @@
+////
```

Comment:
@andrewkroh @adriansr As there is no fields.asciidoc in Filebeat, I assume we keep all the docs for x-pack and the normal beat in one place. I like this. @sayden I would propose to do the same here and for now just have one doc directory under `beats/metricbeat/docs` and the generation includes these already when running `make update` on the metricbeat OSS side.

Reply:
I made a change to the docs_collector.py in filebeat to make this work.

https://github.com/elastic/beats/blob/a07884eceeeb0353e678696e6dabc8edc2eaf2b3/filebeat/scripts/docs_collector.py#L24

I did encounter an issue with the fields docs that still needs to be addressed. The fields that are exclusive to x-pack are not included in the fields-docs.

https://github.com/elastic/beats/blob/a07884eceeeb0353e678696e6dabc8edc2eaf2b3/filebeat/scripts/docs_collector.py#L53-L55